src/test/regress/test_times.log

```diff
@@ -0,0 +1,12 @@
+/pg_regress/multi_cluster_management 0.650
```

Comment:
I think we don't want this file in .git?

Reply:
Yup; Hadi fixed it (added to `gitignore`)